charts/biosdk-service/templates/deployment.yaml

```diff
@@ -94,6 +94,12 @@ spec:
               value: {{ .Values.containerSecurityContext.runAsUser }}
             - name: JDK_JAVA_OPTIONS
               value: {{ .Values.additionalResources.javaOpts }}
+            - name: server_servlet_context_env
```

Comment:
Add the key pair of env variables to values.yaml in extraEnvVars section and consume from there the parameter name and values.

Reply:
updated as requeted